app/scripts/modules/core/src/managed/Loading.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import { Spinner } from 'core/widgets';
+
+export const Loading = ({ message }: { message: string }) => {
+  return (
+    <div style={{ width: '100%' }}>
+      <Spinner size="medium" message={message} />
+    </div>
+  );
+};
```

Comment:
I don't think we need a component for this, it's just a div wrapping an existing component.